deploy/app.py

```diff
@@ -69,6 +69,8 @@
 resource_prefix = app.node.try_get_context('resource_prefix') or 'dataall'
 
 source = app.node.try_get_context('repository_source') or 'codecommit'
+repo_string = app.node.try_get_context('repo_string') or 'awslabs/aws-dataall'
+repo_connection_arn = app.node.try_get_context('repo_connection_arn')
```

Comment:
I am aware that it is not ensured for all the configuration options. But, we are trying to throw errors earlier for users when deploying data.all. Could you add a check here that if the source is `github` and `repo_connection_arn` is None it throws an exception?

Reply:
I have taken a closer look to the code and I think there is a better place to add the check for the github values. In `deploy.stacks.pipeline.PipelineStack.validate_deployment_params` we validate other params, we could also pass the source and connection_arn to this function